src/js/view/controls/controls.js

```diff
@@ -384,6 +384,7 @@ export default class Controls {
         this.controlbar.renderVolume(mute, model.get('volume'));
         this.mute.hide();
         utils.removeClass(this.playerContainer, 'jw-flag-autostart');
+        utils.removeClass(this.playerContainer, 'jw-flag-user-inactive');
```

Comment:
This class should only be changed by the `userActive` and `userInactive` methods. It probably makes more sense to call `userActive()` here so that the class is removed **and** the inactive timer is restarted.